src/components/Modals/AddClientModal.jsx

```diff
@@ -36,6 +37,7 @@ const renderWebId = (username) => {
 
 const AddClientModal = ({ showAddClientModal, setShowAddClientModal }) => {
   const { state, dispatch } = useStatusNotification();
+  const notify = useNotification();
```

Comment:
You could destructure the `addNotification` function directly from the `useNotification` hook without needing to called it from `notifiy.addNotification`.